.github/workflows/build_android.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Android
+on:
+  workflow_dispatch:
+  push:
+    branches: main
+  pull_request:
+    branches: main
+jobs:
+  Build:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 12
+      - name: ktlint
+        if: always()
+        uses: eskatos/gradle-command-action@v1
+        with:
+          build-root-directory: android
+          wrapper-directory: android
+          arguments: ktlintCheck
+      - name: Lint
+        if: always()
+        uses: eskatos/gradle-command-action@v1
+        with:
+          build-root-directory: android
+          wrapper-directory: android
+          arguments: lint
+      - name: Test
+        if: always()
+        uses: eskatos/gradle-command-action@v1
+        with:
+          build-root-directory: android
+          wrapper-directory: android
+          arguments: test
+      - name: Assemble
+        if: always()
+        uses: eskatos/gradle-command-action@v1
+        with:
+          build-root-directory: android
+          wrapper-directory: android
+          arguments: assemble
```

Comment:
It depends on how long GitHub actions are free. For private use MacOS agents are 10 times more expensive than Linux agents. In such a case I would dedicate as little work to MacOS agents as possible.

At the moment, however, we have the luxury of making sure our builds are still compatible on both platforms.

Reply:
My confusion is that we're building for the iOS targets in an action called "Build Android" 🙂

Reply:
Got it. It wasn't clear to me how KMP will be built in the future.
Will it have a separate action? Will it be bundled differently based on the executed platform?

Because of these questions (and the desire to keep these changes small) I decided it would be fine to bundle KMP builds into Build Android's action.

In my opinion, once we have a clearer distinction on what's built when we can expand/split this up a bit more.